calcite-qml/Calcite/calcite.qrc

```diff
@@ -32,10 +32,13 @@
         <file>Slider.qml</file>
         <file>SliderHandle.qml</file>
         <file>SliderGroove.qml</file>
+        <file>SpinBox.qml</file>
         <file>Switch.qml</file>
         <file>TextField.qml</file>
         <file>images/check.svg</file>
         <file>images/caret-double-vertical.svg</file>
         <file>images/line-solid.svg</file>
+        <file>images/chevron-left.svg</file>
+        <file>images/chevron-right.svg</file>
```

Comment:
I saw from previous PR that you prefer these resources to be ordered alphabetically. Do you manually change the ordering? When I use QT creator to add new/existing resources, they are placed like this: at the end.

Reply:
Also, I renamed it by removing the size that is in the original names when pulled from calcite icons. Should we stick with doing so?

Reply:
I also remove the size.
I do reorder manually, it makes maintainability easier in the long run in terms of finding what goes where.